src/Resource/GetAllRate.php

```diff
@@ -12,14 +13,11 @@
 
 class GetAllRate implements ProviderResource
 {
-    private $apiKey;
+    public $config;
```

Comment:
Let `$config` be public and it means we can modify this at any time from this class instance.

How about using the private/protected `$config`?

And we can also add the `setConfig` and `getConfig` to allow us to modify this private/protected `$config` :).